### ISSUE TITLE
SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing false positive

### ISSUE THREAD

**Issue by TRowbotham** [slevomat/coding-standard#1197]
Using PHPCS 3.6.0 and slevomat/coding-standard 7.0.2. The comment "Group 2" seems to be the problem here.

```php
<?php

switch ($foo) {
    case 1:
    case 2:
        echo 'Group 1';

        break;

    // Group 2
    case 3:
    case 4:
        echo 'Group 2';

        break;
}
```
```bash
$ vendor/bin/phpcs --sniffs=SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing example.php
E 1 / 1 (100%)



FILE: example.php
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 5 | ERROR | [x] Expected 1 line after "case", found 0.
   |       |     (SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing.IncorrectLinesCountAfterControlStructure)
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

**Comment by kukulich** [comment]
@TRowbotham What's your configuration for this sniff?

**Comment by TRowbotham** [comment]
Here is the config I'm using.

```xml
<?xml version="1.0"?>
<ruleset name="AcmeProject">
    <config name="installed_paths" value="../../slevomat/coding-standard"/>
    <rule ref="SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing">
        <properties>
            <property name="linesCountBefore" value="1" />
            <property name="linesCountBeforeFirst" value="0" />
            <property name="linesCountAfter" value="1" />
            <property name="linesCountAfterLast" value="0" />
            <property name="controlStructures" type="array">
                <element value="while" />
                <element value="for" />
                <element value="foreach" />
                <element value="if" />
                <element value="do" />
                <element value="try" />
                <element value="switch" />
                <element value="case" />
                <element value="default" />
            </property>
        </properties>
    </rule>
</ruleset>
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/6783711e13ab355e0c872567e0a72a22c71c7b02
